src/app/constants/constants.js

```diff
@@ -13,3 +13,18 @@ export const notes = {
     warning: 'warning'
 
 };
+
+export const range = {
```

Comment:
This is ok, but you could keep this classes local inside the component itself, because you won't use them in other components. If yes, than you can make them global, but I think this is not the case